tests/components/deconz/test_binary_sensor.py

```diff
@@ -88,6 +88,7 @@ async def test_binary_sensors(hass):
 
     vibration_sensor = hass.states.get("binary_sensor.vibration_sensor")
     assert vibration_sensor.state == "on"
+    assert vibration_sensor.attributes["device_class"] == "vibration"
```

Comment:
The `device_class` should not change just because we're upgrading the library. Then we're missing a decoupling in home assistant. There should be eg a dict that maps the home assistant device class constants to the deconz device classes.

Reply:
It's because its stored in the library right now, you want me to move it to hass?

Reply:
No, we just need a decoupling in home assistant so the two APIs are not directly coupled. We should add a dict that explicitly maps the different device classes and put that dict in home assisant.

Reply:
It is the same strings as hass use, so I will create platform specific dicts for this instead

Reply:
Better like this?